PyInstaller/building/api.py

```diff
@@ -720,17 +721,14 @@ def assemble(self):
 
         if is_linux:
             # Linux: append data into custom ELF section using objcopy.
+            # For huge executables, this can take so long that users think it's died. Include a crude progress indicator
+            # so that they know otherwise.
```

Comment:
A left-over comment from previous iteration.

Reply:
Ughh. I'll let this CI run finish then get rid of it.